### PR TITLE
SDK-270 [Node.js SDK] Update README introduction section

### PR DESCRIPTION
**Pull request by RommeroInCountry**
Pull Request Checklist
======================

Please read this for our PR Culture:
------------------------------------
https://incountry.atlassian.net/wiki/spaces/ED/pages/46858869/Pull+requests+culture+Draft

Work relates to:
----------------
Link(s) to Jira Task/User Story/Bug/Epic:
- [SDK-270](https://incountry.atlassian.net/browse/SDK-270)

Code Quality (Before submitting PR)
------------
- [x] Ensure code compiles and passes Sonar Lint.
- [x] Do your best to ensure integration tests pass. If they don't then...
- [x] Remove, ignore, or mark pending any brittle tests.
- [x] Remove commented out code.

Target Release
--------------
- [ ] Please put the target release vehicle this needs to be bundled with.
- [ ] Did you increment the version in package.json? What is the new version?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,77 +89,44 @@ async function createStorage(
...
-const { Storage } = require('incountry');
-const storage = new Storage({
-  apiKey: 'API_KEY',
-  environmentId: 'ENVIRONMENT_ID',
-  endpoint: 'INC_URL',
-  encrypt: true,
-  getSecrets: () => '',
+const { createStorage } = require('incountry');
+const storage = await createStorage({
+  apiKey: '<api_key>',
```
Comment (VladimirLevitskiy) [inline README.md:104]: U think oAuth should be in the first place

File: README.md [hunk 2]
```diff
@@ -71,77 +89,44 @@ async function createStorage(
...
+---
 
-#### oAuth Authentication
+#### oAuth options configuration
 
-The SDK also supports oAuth authentication credentials instead of plain API key authorization. oAuth authentication flow is mutually exclusive with API key authentication - you will need to provide either API key or oAuth credentials.
+The SDK allows to precisely configure oAuth authorization endpoints (if needed). Use this option only if your plan configuration requires so.
 
-Below you can find the example of how to create a storage instance with oAuth credentials (and also provide a custom oAuth endpoint):
+Below you can find the example of how to create a storage instance with custom oAuth endpoints:
```
Comment (VladimirLevitskiy) [inline README.md:116]: Maybe we should uppercase the word 'CUSTOM'?

**Review by VladimirLevitskiy — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -5,24 +5,36 @@
...
------
+## Quickstart guide
+To access your data in InCountry Platform by using NodeJS SDK, you need to create an instance of the Storage class using the createStorage async factory method. You can retrieve the `oauth.clientId`, `oauth.clientSecret` and `environmentId` variables from your dashboard on InCountry Portal.
 
-To access your data in InCountry Platform by using NodeJS SDK, you need to create an instance of the `Storage` class using the `createStorage` async factory method.
+```typescript
+const { createStorage } = require('incountry');
+const storage = await createStorage({
+  environmentId: '<environment_id>',
+    oauth: {
```
Comment (artemgolozubov) [inline README.md:26]: Fix indentation pls

File: README.md [hunk 4]
```diff
@@ -71,77 +83,44 @@ async function createStorage(
...
 
 API Key authorization is being deprecated. The backward compatibility is preserved for the `apiKey` parameter but you no longer can access API keys (neither old nor new) from your dashboard.
 
----
-
-The `oauth.clientId`, `oauth.clientSecret` and `environmentId` variables can be fetched from your dashboard on InCountry Portal.
-
-
-Otherwise you can create an instance of the `Storage` class and run all asynchronous checks by yourself (or skip them at your own risk!).
+Below you can find API Key authorization usage example:
```
Comment (artemgolozubov) [inline README.md:93]: Don't you think it'd be better to point out that API keys are deprecated?

File: README.md (hunk 2 above)
Comment (RommeroInCountry) [reply README.md:110]: why so?

File: README.md (hunk 4 above)
Comment (RommeroInCountry) [reply README.md:92]: @artemgolozubov please note - this text is inside deprecation warning section :)

**Review by artemgolozubov — APPROVED** [review]
LGTM

**Review by igor-alekseev-dev — APPROVED** [review]

**Pull request merged**